mlflow/tracking/fluent.py

```diff
@@ -380,6 +380,17 @@ def start_run(
             # if `log_system_metrics` is not specified, we will check environment variable.
             log_system_metrics = MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING.get()
         if log_system_metrics:
+            # Ensure psutil is installed. It was moved to an optional dependency, as it doesn't
+            # have binary for Arm64 Linux and requires build from CPython which was a headache.
+            if importlib.util.find_spec("psutil") is None:
+                raise MlflowException(
+                    "Failed to start system metrics monitoring as package `psutil` is not "
+                    "installed. Run `pip install psutil --upgrade` to resolve the issue, "
+                    "otherwise you can disable system metrics logging by passing "
+                    "`log_system_metrics=False` to start_run() or setting environment "
+                    "variable MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING to False."
```

Comment:
```suggestion
                    "environment variable MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING to False."
```

Reply:
oh do you mean changing line break?

Reply:
Never mind. We already have `environment`. I missed it.